### PR TITLE
Issue 159 schedule embedded form

### LINKED ISSUE #159: Create  page for /schedule/
Currently https://main--vonage--hlxsites.hlx.page/schedule/ just redirects to the main vonage site.  While we are implementing forms we should setup a page that houses the same form content so that we aren't pulled out of the site by a click on such a button on the landing page hero components.

### PR DESCRIPTION
**Pull request by david-granberg**
Adding support for the embedded-form block to display in a full screen mode as is the case with https://www.vonage.com/schedule/ (which is linked from several contact us buttons throughout the clients page). 
Also cleaned up / improved several other issues with the forms implementation, think that just about everything is in order (minus #163 which is something that should be tackled).

Fix #159 

Test URLs:

- Before: https://main--vonage--hlxsites.hlx.page/unified-communications
- After: https://issue-159-schedule-embedded-form--vonage--hlxsites.hlx.page/unified-communications/

- Before: https://main--vonage--hlxsites.hlx.page/unified-communications/integrations
- After: https://issue-159-schedule-embedded-form--vonage--hlxsites.hlx.page/unified-communications/integrations

- Before: https://main--vonage--hlxsites.hlx.page/unified-communications/features/desktop
- After: https://issue-159-schedule-embedded-form--vonage--hlxsites.hlx.page/unified-communications/features/desktop

- Before: https://main--vonage--hlxsites.hlx.page/schedule
- After: https://issue-159-schedule-embedded-form--vonage--hlxsites.hlx.page/schedule

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -1434,3 +1434,331 @@ header .l3-nav-desktop .right-sec-menu-option:last-child {
...
+  position: fixed;
+  right: 0;
+  width: 100%
+}
+
+.no-header .ecommerce-header {
+  display: none
+}
+
+@media (min-width: 768px) {
```
Comment (rrusher) [inline blocks/header/header.css:1641]: Consolidate all the media queries into the 3 sizes used instead defining a new media query for each selector.

File: blocks/landing-page-hero/landing-page-hero.css [hunk 2]
```diff
@@ -770,19 +779,19 @@ button.vlt-icon-close.form-overlay-close::before {
 }
 
 @media (min-width: 992px) {
-  .form-overlay-overlay-bnsform .vlt-form-title {
+  .form-bnsform .vlt-form-title {
     margin-bottom:0;
   }
 }
 
 @media (min-width: 992px) and (max-width:1024px) {
```
Comment (rrusher) [inline blocks/landing-page-hero/landing-page-hero.css:787]: Don't use max-width. Only use min-width.

File: blocks/landing-page-hero/landing-page-hero.css [hunk 3]
```diff
@@ -658,7 +667,7 @@ button.vlt-icon-close.form-overlay-close::before {
 }
 
 @media (min-width: 1024px) {
```
Comment (rrusher) [inline blocks/landing-page-hero/landing-page-hero.css:669]: Consolidate like media queries.

File: blocks/landing-page-hero/landing-page-hero.js [hunk 4]
```diff
@@ -158,35 +158,37 @@ async function fetchFormThanksElem(thanksUrl) {
...
-  }
 
-  formButton.replaceWith(getFormSection(formButton, styleClass));
+    formButton.replaceWith(getFormSection(formButton, styleClass));
 
-  const formPath = rawLink.substring(rawLink.indexOf('/') + 1);
-  const formWrapper = ctasContent.querySelector(overlayStyles.map((overlayStyle) => `.${overlayStyle.class}`).join(','));
-  const formContent = fetchFormContent(`${formPath}`, formWrapper);
-  formWrapper.append(createFormOverlay(await formContent));
+    const formPath = rawLink.substring(rawLink.indexOf('/') + 1);
```
Comment (rrusher) [inline blocks/landing-page-hero/landing-page-hero.js:176]: This is not as clear or reliable as using the URL object.
const formPath = new URL(rawLink.href).pathname;

**Review by dkuntze — APPROVED** [review]

**Pull request merged**